pkg/commands/git.go

```diff
@@ -561,6 +561,16 @@ func (c *GitCommand) GetRemoteURL() string {
 	return utils.TrimTrailingNewline(url)
 }
 
+// CheckRemoteBranchExists Returns remote branch
+func (c *GitCommand) CheckRemoteBranchExists(branch *Branch) bool {
+	_, err := c.OSCommand.RunCommandWithOutput(fmt.Sprintf(
+		"git show-ref --verify -- \"refs/remotes/origin/%s\"",
```

Comment:
I don't think we need the quotation marks here as branches can't have spaces in the name, and we've had issues previously with how quotes are interpreted on windows etc.

Reply:
Fixed.